kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Coordinator.java

```diff
@@ -127,6 +127,7 @@ private boolean receive(Envelope envelope) {
 
   private void commit(boolean partialCommit) {
     try {
+      LOG.info("Processing commit after responses for {}, isPartialCommit {}",commitState.currentCommitId(), partialCommit);
```

Comment:
nit: I feel this one is unnecessary given that we the commit message for `Commit {} ready, received responses for all {} partitions`.

Reply:
There is a flow where you don't get here and I wanted to capture it